src/ceph-disk/ceph_disk/main.py

```diff
@@ -2377,13 +2379,14 @@ def set_or_create_partition(self):
                       self.args.lockbox)
             self.partition = self.create_partition()
 
-    def create_key(self):
+    def create_key(self, cluster):
         key_size = CryptHelpers.get_dmcrypt_keysize(self.args)
         key = open('/dev/urandom', 'rb').read(key_size / 8)
         base64_key = base64.b64encode(key)
         command_check_call(
             [
                 'ceph',
+                '--cluster', cluster,
```

Comment:
you would be able to just use that `self.args.cluster` in here, which already defaults to `ceph` if not explicitly defined